main.go

```diff
@@ -1,50 +1,40 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 
 	"net"
 
 	"github.com/nov1n/kubernetes-workflow/pkg/client"
-	"github.com/nov1n/kubernetes-workflow/pkg/job"
 
 	k8sApi "k8s.io/kubernetes/pkg/api"
```

Comment:
Why do you do this?
I think we should only use this when there is a collision with our own api package.